cmd/storagenode-updater/main.go

```diff
@@ -112,10 +119,15 @@ func cmdRun(cmd *cobra.Command, args []string) (err error) {
 	}()
 
 	loopFunc := func(ctx context.Context) (err error) {
-		if err := update(ctx, ident.ID); err != nil {
+		if err := update(ctx, runCfg.BinaryLocation, runCfg.ServiceName, renameStoragenode); err != nil {
 			// don't finish loop in case of error just wait for another execution
 			log.Println(err)
 		}
+
+		if err := update(ctx, os.Args[0], updaterServiceName, renameUpdater); err != nil {
```

Comment:
What is the value of `os.Args[0]` in this case?

Reply:
binary name, I will assign it to variable